### PR TITLE
add aurora TVL

### PR DESCRIPTION
**Pull request by oleksiivinogradov**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/MultiSwap: 
        
 
  
    
    
    Error: 
        Found export keys that were not part of specification: tvlAurora
    
        List of valid keys: 
    				
    				tvl
    				staking
    				methodology
    				pool2
    				misrepresentedTokens
    				fetch
    				timetravel
    				borrowed
    				start
    				masterchef
    				doublecounted
    				treasury
    				hallmarks
    				incentivized
    				offers
    				deadFrom
    				broken
    				ownTokens

File: projects/MultiSwap/index.js [hunk 1]
```diff
@@ -111,8 +116,46 @@ async function tvl(_, _b, { [chain]: block }) {
...
+  delete kavaBal[customKavaKey]
+  return kavaBal
+}
+
 module.exports = {
   kava: {
     tvl
+  },
+  aurora: {
+    tvlAurora
```
Comment (g1nt0ki) [inline projects/MultiSwap/index.js:159]: change this to `tvl: tvlAurora`

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/MultiSwap/index.js (hunk 1 above)
Comment (oleksiivinogradov) [reply projects/MultiSwap/index.js]: done

**Comment by llamatester** [comment]
Error while running adapter at projects/MultiSwap: 
        
 
  
    
    
    Error: call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="poolsCount()", data="0x", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.6.4)
    Truncated error stack:
        at Interface.decodeFunctionResult (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/abi/lib/interface.js:388:23)

**Comment by oleksiivinogradov** [comment]
fixed

**Comment by llamatester** [comment]
The adapter at projects/MultiSwap exports TVL: 
        
 
 
    aurora                    0.00000767186748
    kava                      0.07572396235893525
    
    total                    0.07573163422641525

**Pull request merged**